### DISCUSSION TITLE
Per-group loopback ranges

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#1328] [Features]
Suppose I have a topology with 2 groups, one representing nodes in data center 1 and a second for data center 2

I'd like to assign loopback IPs that are specific per data center: e.g. 10.1.x.y for DC1, and 10.2.x.y for DC2.
How could I do this?

**Comment by ipspace** [comment]
I don't think you can (today). We would need **loopback.pool** attribute.

**Comment by ipspace** [comment]
Loopback interface can have **pool** attribute (https://netlab.tools/nodes/#node-loopback)
